wiki/Beatmap_Editor/Menu/en.md

```diff
@@ -1,441 +1,256 @@
-</tr>
-<tr class="odd">
-<td><p>Upload Beatmap...<br />
-(Ctrl + Shift + U)</p></td>
-</tr>
-<tr class="even">
-<td><p>Export Package...</p></td>
-</tr>
-<tr class="odd">
-<td><p>Extract Map Package</p></td>
-</tr>
-<tr class="even">
-<td><p>Import from...</p></td>
-</tr>
-<tr class="odd">
-<td><p>colspan=2| Programmer's commands</p></td>
-</tr>
-<tr class="even">
-<td><p>Open Song Folder</p></td>
-</tr>
-<tr class="odd">
-<td><p>Open .osu in Notepad</p></td>
-</tr>
-<tr class="even">
-<td><p>Open .osb in Notepad</p></td>
-</tr>
-<tr class="odd">
-<td><p>colspan=2| Emergency Exit command.</p></td>
-</tr>
-<tr class="even">
-<td><p>Exit...<br />
-(Esc)</p></td>
-</tr>
-</tbody>
-</table>
+------
+
+![File menu](M_File.jpg "File menu")
+
+**Basic File commands.**
+
+### File commands
+
+| Name | Description |
+| ---- | ----------- |
+| Clear All Notes (Ctrl + N) | Purge the current difficulty's hitnotes. |
+| Open Difficulty... | **For Editing:** Quickly switch between difficulties. **For Reference:** (Mania Only) Overlay the selected difficulty over the (currently) opened one. |
+| Save (Ctrl + S) | Save current changes (both .osu and .osb) |
+| Create a new Difficulty... | Save the current changes on this difficulty as a new difficulty. The old difficulty is preserved to its last saved state when reloaded. |
+
+### Revert commands
+
+| Name | Description |
+| ---- | ----------- |
+| Revert to Saved (Ctrl + L) | Cancel current changes and return to the last saved state. For general mishaps (hitnotes). |
+| Revert to Saved (Full) (Ctrl + Shift + L) | Cancel current changes and return to the last saved state. For complicated mishaps (timing, storyboard). |
+
+### Troubleshoot commands
+
+| Name | Description |
+| ---- | ----------- |
+| Test Beatmap (F5) | Test the Beatmap. Current changes will be saved in order to test. |
+| Open AiMod (Ctrl + Shift + A) | Open AiMod. |
+
+### Beatmap Packaging commands
+
+| Name | Description |
+| ---- | ----------- |
+| Upload Beatmap... (Ctrl + Shift + U) | Upload the beatmap to the [WIP section](http://osu.ppy.sh/forum/10). Requires osu!account logged in to the website to view "Beatmaps" forum. |
+| Export Package... | Export the beatmap as .osz. The file will be sent to "Exports" folder and you will be directed to osu!/Exports from the Beatmap Editor. |
+| Extract Map Package | ? |
+| Import from... | **bms/bme:** (osu!mania only) Open .bms/.bme file to be imported as osu!mania difficulty. |
+
+### Programmer's commands
+
+| Name | Description |
+| ---- | ----------- |
+| Open Song Folder | Open the song folder location inside osu!/Songs. |
+| Open .osu in Notepad | Open beatmap's difficulty file in MS Notepad. |
+| Open .osb in Notepad | Open base file (storyboard) in MS Notepad. |
+
+### Emergency Exit command
+
+| Name | Description |
+| ---- | ----------- |
+| Exit... (Esc) | Exit Beatmap Editor. May prompt you to save recent changes. Sometimes *ignores* Design tab changes. |
```

Comment:
Just to say, you _could_ also press `Alt` + `F4` to exit the beatmap editor (may prompt to save tho)